vllm/attention/layer.py

```diff
@@ -179,9 +182,28 @@ def __init__(
             ).parallel_config.pipeline_parallel_size)
         ]
 
-        self.q_range = torch.tensor(envs.Q_SCALE_CONSTANT, dtype=torch.float32)
-        self.k_range = torch.tensor(envs.K_SCALE_CONSTANT, dtype=torch.float32)
-        self.v_range = torch.tensor(envs.V_SCALE_CONSTANT, dtype=torch.float32)
+        try:
+            self.q_range = torch.tensor(envs.Q_SCALE_CONSTANT,
+                                        dtype=torch.float32)
+            self.k_range = torch.tensor(envs.K_SCALE_CONSTANT,
+                                        dtype=torch.float32)
+            self.v_range = torch.tensor(envs.V_SCALE_CONSTANT,
+                                        dtype=torch.float32)
+        except RuntimeError as e:
+            if torch.cuda.is_available():
+                # This helps to see how much memory is allocated when using CUDA
+                logger.error(
+                    "Failed to initialize attention q/k/v range "
+                    "constants: %s", e)
+                logger.debug("CUDA device: %s", torch.cuda.current_device())
+                allocated_gb = torch.cuda.memory_allocated() / 1024**3
```

Comment:
```suggestion
                allocated_gb = torch.cuda.memory_allocated() / GiB_bytes
```

Can import `GiB_bytes` from `vllm.utils` to make this more readable